hack/verify-api-descriptions.sh

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+if [ "$#" -lt 1 ]; then
+	echo "Usage: verify-api-descriptions.sh <OpenShift API version>"
+    exit
+fi
+
+OS_ROOT=$(dirname "${BASH_SOURCE}")/..
+source "${OS_ROOT}/hack/common.sh"
+
+cd "${OS_ROOT}"
+
+DEF_MISSING=false
+TYPES_LOCATIONS=$(grep -Po "(?<=\_ \")[A-Za-z0-9\.\/]+(?=\")" pkg/api/${1}/register.go)
```

Comment:
Not all types are directly used by the user and grepping through a sort file is nasty.  Try walking the swagger.  We'll have to do that for other work anyway.